src/util/permissions.tsx

```diff
@@ -342,3 +367,21 @@ export const enablePermissionsFeature = (): boolean => {
 
   return hasAccessManagement && (hasOIDCSettings || userShowPermissions);
 };
+
+// each resource type has specific columns to display, which should uniquely identify the resource
+export const getResourceOptionColumns = (type: string) => {
+  const resourceOptionColumns: Record<string, (keyof ResourceDetail)[]> = {
+    image: ["aliases", "fingerprint", "description", "imageType"],
```

Comment:
I think the description should come first, as it will be the label on the toggle after selection.